app/models/service_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ServiceUpdate
+  CLAIMS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/claims_service_updates.yml").freeze
+  PLACEMENTS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/placements_service_updates.yml").freeze
+
+  include ActiveModel::Model
+
+  attr_accessor :date, :title, :content
+
+  def id
+    title.parameterize
+  end
+
+  def self.where(service:)
+    path = yaml_file(service:)
+    updates = YAML.load_file(path) if path
+
+    return {} unless updates
+
+    updates.map { |params| new(params) }
```

Comment:
You are coding very defensively, which is a very good thing when you are consuming something changeable or depending on asynchronous jobs. But we don't want to code defensively against things we are responsible for (ie, our file structure). Our defence against someone deleting a file should be the tests. In this situation, if someone deletes the file, the tests would pass (because of the early returns), but the feature would be broken. 

And a warning, I'm pedantic about naming -- I don't think these are `params`, they are individual updates. And the `yaml_file` function returns a path, not a `yaml_file`

There are lots of ways to write this, but one alternative would be:

```suggestion
    path = file_path(service:)
    updates = YAML.load_file(path) || []

    updates.map { |service_update| new(service_update) }
```